.gitignore

```diff
@@ -8,4 +8,12 @@ container/*/sql
 .DS_Store
 .vscode
 
-frontend/public/bundle.js
+frontend/public
```

Comment:
Why is this in the gitignore? The base of our app index.html is literally inside there.